takwimu/profiles/census.py

```diff
@@ -56,7 +56,7 @@ def get_population(geo, session):
         'is_missing': is_missing,
         'sex_dist': sex_dist,
         'residence_dist': residence_dist,
-        'total_dist': total_dist,
+        'total_population': total_dist,
     }
```

Comment:
Can we add [population density](https://wazimap.co.za/profiles/province-LIM-limpopo/) (people per square kilometre) as well ?